Traceback (most recent call last):
  File "C:\Users\Kinkintama\Documents\GitHub\EasyChat-App\main.py", line 16, in <module>
    EasyChatApp().run()
  File "C:\Users\Kinkintama\anaconda3\lib\site-packages\kivy\app.py", line 955, in run
    runTouchApp()
  File "C:\Users\Kinkintama\anaconda3\lib\site-packages\kivy\base.py", line 574, in runTouchApp
    EventLoop.mainloop()
  File "C:\Users\Kinkintama\anaconda3\lib\site-packages\kivy\base.py", line 339, in mainloop
    self.idle()
  File "C:\Users\Kinkintama\anaconda3\lib\site-packages\kivy\base.py", line 383, in idle
    self.dispatch_input()
  File "C:\Users\Kinkintama\anaconda3\lib\site-packages\kivy\base.py", line 334, in dispatch_input
    post_dispatch_input(*pop(0))
  File "C:\Users\Kinkintama\anaconda3\lib\site-packages\kivy\base.py", line 302, in post_dispatch_input
    wid.dispatch('on_touch_up', me)
  File "kivy\_event.pyx", line 731, in kivy._event.EventDispatcher.dispatch
  File "C:\Users\Kinkintama\anaconda3\lib\site-packages\kivy\uix\behaviors\button.py", line 179, in on_touch_up
    self.dispatch('on_release')
  File "kivy\_event.pyx", line 727, in kivy._event.EventDispatcher.dispatch
  File "kivy\_event.pyx", line 1307, in kivy._event.EventObservers.dispatch
  File "kivy\_event.pyx", line 1191, in kivy._event.EventObservers._dispatch
  File "C:\Users\Kinkintama\anaconda3\lib\site-packages\kivy\lang\builder.py", line 55, in custom_callback
    exec(__kvlang__.co_value, idmap)
  File "C:\Users\Kinkintama\Documents\GitHub\EasyChat-App\libs\uix\kv\login_screen.kv", line 47, in <module>
    on_release: root.sign_in(username.text, password.text)
  File "C:\Users\Kinkintama\Documents\GitHub\EasyChat-App\libs\uix\baseclass\login_screen.py", line 65, in sign_in
    home_screen.receive()
  File "C:\Users\Kinkintama\Documents\GitHub\EasyChat-App\libs\uix\baseclass\home_screen.py", line 38, in receive
    is_exist, message, sender = self.check_notification(chats['id'])
  File "C:\Users\Kinkintama\Documents\GitHub\EasyChat-App\libs\uix\baseclass\home_screen.py", line 93, in check_notification
    if resp['notification']:
KeyError: 'notification'
